dist/docker/Dockerfile

```diff
@@ -46,6 +46,9 @@ RUN apt-get update -y && \
   # Clean up the lists work
   rm -rf /var/lib/apt/lists/*
 
+# Install Helm
+RUN curl -L https://git.io/get_helm.sh | bash
```

Comment:
I just realized this is missing the `client-only` init. Not sure if helm is truly usable in this state.

Reply:
I think that should be in the entrypoint if we add it in, otherwise you'd still have to run it if the repo cache is old.

Reply:
I do agree that it makes more sense in the entrypoint. I do believe helm 3 is getting rid of the requirement because there is no tiller, but until the script installs helm 3 we probably need this.

Reply:
I am worried that if we add it in now the entrypoint will be broken for a day or two until someone removed the client-side init.

I think we can add the init to the entrypoint but we should make sure it ignores a failed command so that the container still works when helm 3 comes out.

Reply:
I think we should add detection for v2 and v3. If it's v2, run the command, and if it's not, then don't.